global-api/migrations/versions/88e4463fef06_locode_data_table.py

```diff
@@ -33,8 +33,8 @@ def upgrade() -> None:
         sa.Column("activity_value", sa.Float, nullable=True),
         sa.Column("activity_units", sa.String, nullable=True),
         sa.Column("gas_name", sa.String, nullable=False),
-        sa.Column("emission_factor_value", sa.Float, nullable=False),
-        sa.Column("emission_factor_units", sa.String, nullable=False),
+        sa.Column("emission_factor_value", sa.Float, nullable=True),
+        sa.Column("emission_factor_units", sa.String, nullable=True),
```

Comment:
Editing already executed migrations will not make them re-run on the server, so please write a new migration that changes these columns instead of editing the existing migration